server/apps/immich/src/modules/schedule-tasks/schedule-tasks.service.ts

```diff
@@ -94,7 +94,11 @@ export class ScheduleTasksService {
       });
 
       for (const exif of exifInfo) {
-        await this.metadataExtractionQueue.add(reverseGeocodingProcessorName, { exif }, { jobId: randomUUID() });
+        await this.metadataExtractionQueue.add(
+          reverseGeocodingProcessorName,
+          { exifId: exif.id, latitude: exif.latitude!, longitude: exif.longitude! },
```

Comment:
Eslint complains here image.png

Reply:
It's complaining because the type system in typeorm isn't dynamic, so it doesn't realise none of the results for latitude and longitude can be null due to the where clause we define in the job. I personally think we shouldn't fail the build on warnings as they are that, warnings. If we want certain rules to fail builds they should be errors, warnings should be allowed through. For this specific case we can add a comment to ignore the rule, my linter in my IDE wasn't working so I couldn't auto-complete the ignore comment annoyingly.